### PR TITLE
fixing integration test

### PR DESCRIPTION
**Pull request by AndrewLitteken**
adding forgotten line to run without swap mirroring.

Fixes https://github.com/SupertechLabs/qiskit-superstaq/issues/213

### CODE REVIEW

**Comment by vtomole** [comment]
Still fails https://github.com/SupertechLabs/qiskit-superstaq/runs/7998736841?check_suite_focus=true

File: qiskit_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -119,12 +119,13 @@ def test_qscout_compile_swap_mirror(provider: qss.SuperstaQProvider) -> None:
...
 
+    out = provider.qscout_compile(qc, mirror_swaps=False)
     op = qiskit.quantum_info.Operator(out.circuit)
     expected_op = qiskit.quantum_info.Operator(qc)
     assert op.equiv(expected_op)
 
     num_two_qubit_gates = 0
-    for op in qc:
+    for op in out.circuit:
         if len(op.qubits) > 1:
```
Comment (richrines1) [inline qiskit_superstaq/daily_integration_test.py:129]: should be something like
```suggestion
    for inst, _, _ in out.circuit:
        if inst.num_qubits == 2:
```

**Comment by AndrewLitteken** [comment]
I think this fix should fix that type error

**Review by richrines1 — APPROVED** [review]
lgtm! (confirmed it passes locally)

**Review by vtomole — APPROVED** [review]
LGTM

**Pull request merged**